src/services/health-check/default-tagline/health-check-default-tagline-presenter.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Yoast\WP\SEO\Services\Health_Check\Default_Tagline;
+
+use Yoast\WP\SEO\Services\Health_Check\Health_Check_Result_Builder;
+
+/**
+ * Health_Check_Default_Tagline_Presenter
+ */
+class Health_Check_Default_Tagline_Presenter {	
+
+	/**
+	 * test_name
+	 *
+	 * @var string
+	 */
+	private $test_identifier;
+	
+	/**
+	 * __construct
+	 *
+	 * @param  string $test_identifier
+	 * @return void
+	 */
+	public function __construct($test_identifier) {
+		$this->test_identifier = $test_identifier;
+	}
+	
+	/**
+	 * get_success_result
+	 *
+	 * @return string[]
+	 */
+	public function get_success_result() {
+		$result_builder = $this->get_result_builder();
+		return $result_builder
+			->set_label(esc_html__( 'You changed the default WordPress tagline', 'wordpress-seo' ))
+			->set_status_good()
+			->set_description(esc_html__( 'You are using a custom tagline or an empty one.', 'wordpress-seo' ))
+			->build();
+	}
+	
+	/**
+	 * get_has_default_tagline_result
+	 *
+	 * @return string[]
+	 */
+	public function get_has_default_tagline_result() {
+		$result_builder = $this->get_result_builder();
+		return $result_builder
+			->set_label(esc_html__( 'You should change the default WordPress tagline', 'wordpress-seo' ))
+			->set_status_recommended()
+			->set_description(esc_html__( 'You still have the default WordPress tagline. Even an empty one is probably better.', 'wordpress-seo' ))
+			->set_actions($this->get_actions())
+			->build();
+	}
+	
+	/**
+	 * get_result_builder
+	 *
+	 * @return Health_Check_Result_Builder
+	 */
+	private function get_result_builder() {
+		$result_builder = new Health_Check_Result_Builder();
```

Comment:
this is abstracted away neatly. However, calling a constructor makes it very hard to unit test. As an exercise, what's needed (what pattern) to use dependency injection, so we can prevent the constructor call?